x/bank/client/cli/tx.go

```diff
@@ -60,3 +68,76 @@ func NewSendTxCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewMultiSendTxCmd returns a CLI command handler for creating a MsgMultiSend transaction.
+// For a better UX this command is limited to send one coin from one address to multiple addresses.
+func NewMultiSendTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "multi-send [from_key_or_address] [to_address_1, to_address_2, ...] [amount]",
+		Short: "Send funds from one account to multiple others.",
+		Long: `Send funds from one account multiple others.
+		By default, sends the [amount] to each address of the list.
+		Using the '--split' flag, the [amount] is split equally between the addresses.
+		Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
+		When using '--dry-run' a key name cannot be used, only a bech32 address.`,
```

Comment:
```suggestion
		Long: `Send funds from one account multiple others.
By default, sends the [amount] to each address of the list.
Using the '--split' flag, the [amount] is split equally between the addresses.
Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
When using '--dry-run' a key name cannot be used, only a bech32 address.
`,
```

Reply:
Use `make build` and then `./build/simd tx bank ...` to see what this command actually looks like to test :)